server/liveblog/themes/themes_assets/classic/main.js

```diff
@@ -16,6 +16,11 @@
         function retrieveUpdate() {
             return vm.pagesManager.retrieveUpdate(true);
         }
+        var srcset = '';
+        angular.forEach(config.blog.picture.renditions, function(value) {
+            srcset += ', ' + value.href + ' ' + value.width + 'w';
```

Comment:
the explicit thoughts are there, size and ratio are not a good idea because browser will not know which ones should it peek. also I don't know if that is valid syntax.